holoviews/plotting/bokeh/renderer.py

```diff
@@ -306,6 +329,8 @@ def load_nb(cls, inline=True):
         """
         Loads the bokeh notebook resources.
         """
-        from bokeh.io.notebook import curstate
+        LOAD_MIME_TYPE = bokeh.io.notebook.LOAD_MIME_TYPE
+        bokeh.io.notebook.LOAD_MIME_TYPE = MIME_TYPES['jlab-hv-load']
```

Comment:
Is this something bokeh expects you to do?

Reply:
No, it's me temporarily switching which JupyterLab extension to publish to. It means you won't have to install both the bokeh and the holoviews extension.

Reply:
Ok. That sounds good as long as we feel this approach will be robust enough.